apis/token.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+var DocumentDBClient = require('documentdb').DocumentClient;
+var http = require('http');
+var request = require('request');
+
+/*
+sendEmailInvite(email: string, personId: string) {
+        if (email != null) {
+            console.log("Make it to send shit");
+            console.log("Email: " + email);
+            console.log("Person Id: " + personId);
+            // What to do with the response
+            var response = this.http.post("https://mscs-cf-functions.azurewebsites.net/api/notify_police?code=9hDa/Q8w5UX69H3WaxQPR5jdGOEok6Vub2PIbKpEHawJda19TfqIeg==",
+                {
+                    'email': email,
+                    'userId': personId
+                })
+                .map(response => response.json())
+                .toPromise()
+                .then(res => {
+                    console.log(res);
+                    return res;
+                });
+        }
+        else {
+            console.log("Couldn't send anything. Email was null!")
+        }
+    }
+*/
+
+
+
+var Token = function() {
+}
+
+Token.prototype.send = function(id, email) 
+{
+    return new Promise((resolve, reject) => {
+        var r;
+        request.post(
+        'https://mscs-cf-functions.azurewebsites.net/api/notify_police?code=9hDa/Q8w5UX69H3WaxQPR5jdGOEok6Vub2PIbKpEHawJda19TfqIeg==',
```

Comment:
there is a config value for that now so it doesn't have to be hard coded `config.notifyPoliceUrl`